src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```
@@ -2388,7 +2389,9 @@ because xbuild doesn't support framework reference assemblies.
 
 <PropertyGroup>
   <_GeneratePackageManagerJavaDependsOn>
+    $(BeforeGenerateAndroidManifest);
     _GenerateJavaStubs;
+		$(AfterGenerateAndroidManifest);
```

I think this line has tabs: 
```suggestion
    $(AfterGenerateAndroidManifest);
```

---

This is why it's handy to just edit the file.